containers/jetty/src/main/java/com/yahoo/athenz/container/log/JettyConnectionLogger/ConnectionLogEntry.java

```diff
@@ -0,0 +1,195 @@
+/*
+ *
+ *  * Copyright 2020 Verizon Media
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.yahoo.athenz.container.log.JettyConnectionLogger;
+
+import java.time.Instant;
+import java.util.Optional;
+import java.util.UUID;
+
+public class ConnectionLogEntry {
+    private final UUID id;
+    private final Instant timestamp;
+    private final Double durationSeconds;
+    private final String peerAddress;
+    private final Integer peerPort;
```

Comment:
We only record failed connections so some attributes used in Vespa solution were removed as they are only valid for successfully SSL handshakes (such bytes received / number of requests / agreed protocol etc...)